tests/tom-swifty-test/SwiftReflector/MetatypeTests.cs

```diff
@@ -229,6 +230,7 @@ public static void Main (string[] args)
 
 				var output = TestRunning.Execute (temp.DirectoryPath, "TestIt.exe", PlatformName.macOS);
 				Assert.AreEqual (expected, output);
+				var typeBasedClassName = typeName.Replace('.', '_');
```

Comment:
same (style)
maybe you should add (if not already there) the `.editorconfig` file to the repo ?

Reply:
Will add this on my todos :)